proto/httpcache.proto

```diff
@@ -10,14 +10,14 @@ import "google/api/httpbody.proto";
 import "google/api/annotations.proto";
 
 message _HttpGetRequest {
-  bytes cache_name = 1;
-  bytes cache_key = 2;
+  string cache_name = 1;
+  string cache_key = 2;
   string token = 3;
 }
 
 message _HttpSetRequest {
-  bytes cache_name = 1;
-  bytes cache_key = 2;
+  string cache_name = 1;
+  string cache_key = 2;
```

Comment:
What's envoy doing here? Is it base64 encoding?

Do we require that these chars `/` `=` and `+` are percent-encoded?

Reply:
With strings this will only support url path-safe cache names and cache keys. Values are in the body so it _should_ support arbitrary values. Still working on making sure about that part.

Reply:
Oh I see there's no base64 encoding here for cache_name/cache_key.

Reply:
Right, thanks!